baton-maven-plugin/src/main/java/org/technologybrewery/baton/GroupSummary.java

```diff
@@ -0,0 +1,29 @@
+package org.technologybrewery.baton;
+
+import java.util.ArrayList;
+import java.util.List;
+
```

Comment:
S: I know it seems obvious, but sometimes writing the comment identifies how names may or may not make sense.  Our goal is for all non getter/setter public classes and methods to have a comment.

Reply:
Added comments.